### ISSUE TITLE
MainViewController.ViewDidDisappear

### ISSUE THREAD

**Issue by juanlaube** [toggl/mobileapp#5463]
## 💥 Crash Report
New crash in iOS 2.0. Probably related to the tap to edit onboarding step.

### 👥 Impact
Happened 4 times so far in iOS 2.0.

### 🛰 Link
https://appcenter.ms/orgs/Toggl/apps/Toggl-iOS/crashes/errors/2386828140u/overview

### 📝  Exception & Stack Trace
`SIGABRT: Object reference not set to an instance of an object`
```cs
MainViewController.ViewDidDisappear (System.Boolean animated)
(wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
UIApplication.Main (System.String[] args, System.IntPtr principal, System.IntPtr delegate)
UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName)
Application.Main (System.String[] args)
```

**Comment by juanlaube** [comment]
The fix didn't work and this is still happening in 2.0.1.

**Comment by juanlaube** [comment]
There's a PR merged in `develop` already (#5616) that should fix this. We can reopen the issue if the crash still happens after the iOS 2.2 release.
